leader-election-impl/src/main/java/com/palantir/paxos/PaxosQuorumChecker.java

```diff
@@ -236,7 +236,7 @@ private PaxosQuorumChecker() {
         }, OUTSTANDING_REQUEST_CANCELLATION_TIMEOUT_MILLIS, TimeUnit.MILLISECONDS);
 
         if (shouldLogCancellationStatus()) {
-            log.warn("Quorum checker canceled pending requests"
+            log.debug("Quorum checker canceled pending requests"
```

Comment:
Does it make sense to add a `log.isDebugEnabled()` above? I don't know how `getOneMinuteRate()` is implemented, but if that is actually costly then we should avoid calculating it if we don't have to

Reply:
Was there an offline discussion related to this question?

Reply:
I have changed it to check `log.isDebugEnabled()`. Metrics shouldn't be costly, still checking if debug enabled is easy enough. Also, it prevents generating a random number every time (by checking `isDebugEnabled` before `shouldLog` which generates a random value to sample logs)

Reply:
cool, thanks!